### ISSUE TITLE
Incorrect return for log plot inputs

### ISSUE THREAD

**Issue by sco1** [sco1/coordinate2normalized#6]
Normalized coordinates returned by `coord2norm` are not correct with log plot inputs. For example:

```
myaxes = axes();
x = logspace(-1,2);
y = exp(x);
loglog(x,y)

[normx, normy] = coord2norm(myaxes, [x(40) x(41)], [y(40) y(41)]);
annotation('arrow', normx, normy);
```

Returns the following:

```
>> [x(40) x(41)]

ans =

   24.4205   28.1177

>> [y(40) y(41)]

ans =

   1.0e+12 *

    0.0403    1.6269

>> [normx; normy]

ans =

    0.3187    0.3474
    0.1100    0.1100
```

![asdf](https://cloud.githubusercontent.com/assets/5323929/11402538/6cf46cb0-9366-11e5-9721-77109d1a8a9e.png)

Using R2015b

**Comment by sco1** [comment]
Issues with `coord2norm` handling of `loglog`, `semilogx`, and `semilogy` resolved in 6d0b42b

Utilizing the same test case:
![untitled](https://cloud.githubusercontent.com/assets/5323929/11403691/3c4a7ff4-936c-11e5-90a7-3b828726e7be.png)

**Comment by sco1** [comment]
I suck at math 01fa30b

**Comment by sco1** [comment]
Issues with norm2coord handling of loglog, semilogx, and semilogy resolved in 02a9a09

Utilizing the same test case:

![untitled](https://cloud.githubusercontent.com/assets/5323929/11516447/3e23a3a2-9851-11e5-8765-aeb388664599.png)
